controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -165,27 +165,30 @@ def test_create_snapshot_with_get_array_type_exception(self, storage_agent):
         self._test_create_object_with_get_array_type_exception(storage_agent)
 
     @patch("controller.controller_server.csi_controller_server.get_agent")
-    def test_create_snapshot_get_snapshot_exception(self, storage_agent):
+    def get_snapshot_returns_error(self, storage_agent, exception, grpc_status, message):
         storage_agent.return_value = self.storage_agent
-        self.mediator.get_snapshot.side_effect = [Exception("error")]
+        self.mediator.get_snapshot.side_effect = [exception]
 
         self.servicer.CreateSnapshot(self.request, self.context)
 
-        self.assertEqual(self.context.code, grpc.StatusCode.INTERNAL)
-        self.assertTrue("error" in self.context.details)
+        self.assertEqual(self.context.code, grpc_status)
+        self.assertTrue(message in self.context.details)

```

Comment:
could you change to [`assertIn`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn) while you're at it?

Reply:
done